### DISCUSSION TITLE
DreamBerd is raising a pre-seed

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#427] [Announcements]
We tried to keep this one under wraps but the _hype is too strong_.

**The rumours are true! DreamBerd is raising a pre-seed round.**

We already have some big players on the table.

• £10 - Sunil Pai (creator of [partykit](https://www.partykit.io/))
• £10 - Alex Dytrych (creator of [infinite scroll](https://alex.dytry.ch/toys/infinite-scroll/))
• $35 - David Sheldrick (creator of [patch-package](https://github.com/ds300/patch-package))
• sweet wrapper[^1] - Steve Ruiz (creator of [tldraw](https://tldraw.com))
• £1 - Lu Wilson (creator of [DreamBerd](https://github.com/TodePond/DreamBerd))

If you're interested in investing in DreamBerd, feel free to book a call via my [calendly](https://calendly.com/DreamBerd). But move fast - it's likely to fill up quickly. That's how hot DreamBerd is right now! 🔥

Or if you're already convinced, simply head over to [dreamberd.computer](http://dreamberd.computer), and pick an amount that works for you. Then enjoy your new life as a DreamBerd investor!

**Remember:** Being a good investor isn't about making money. It's about having the best portfolio. Add DreamBerd to your arsenal *today*!

## A message from our founder

> Hi, it's Lu Wilson here, CEO of the DreamBerd Foundation.
>
> Your valuable investment will be used to ensure the continued development of DreamBerd, the perfect programming language.
> 
> Unfortunately, maintaining DreamBerd has become a costly task. The language has become so popular that I now receive thousands of [incomplete DreamBerd programs by email](https://github.com/TodePond/DreamBerd/tree/main#ai) everyday. This has become too much for me to handle alone.
> 
> _With your investment money, I'll be able to tackle the growing backlog of emails by paying for Replit's child labour services!_
> 
> Thanks, and thanks for tuning into this week's DreamBerd update.

[^1]: sweet wrapper by verbal agreement only

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/v%24.%24'>DreamBerd is raising a pre-seed</a>.</em>

**Comment by NotAShelf** [comment]
can I pledge £1 now and pay later in installations

**Reply by TodePond** [reply]
sure, just manually reinvest every month

**Reply by NotAShelf** [reply]
@TodePond I've sent this month's £0.01, please confirm so that I know the system is working and can be used over the course of next 99 months before I've achieved a donation of full £1

**Comment by mybearworld** [comment]
Can I pay £-1?

**Reply by TodePond** [reply]
Yes but please be careful 👷‍♀️

**Comment by ivyreese** [comment]
You're TodePond??!!

**Reply by mybearworld** [reply]
No, I'm not TodePond.

**Reply by TodePond** [reply]
like usual, thanks for answering all of these for me @mybearworld

**Comment by stackmator** [comment]
Believing is investing 🚀

**Comment by radeeyate** [comment]
can i donate $3.456e11 USD?

**Reply by TodePond** [reply]
Sorry I only accept great british quid

**Reply by radeeyate** [reply]
shoot, £3.456e11 then?

**Comment by tylerhasman** [comment]
Can I get cash back on my donation?

**Reply by TodePond** [reply]
Not in the traditional sense, but the cash will trickle back to you in the end

**Reply by tylerhasman** [reply]
Sounds fair to me

**Comment by riley-martine** [comment]
Since our whole multi-million dollar company is built on Dreamberd, I tried to get us to sponsor you. Unfortunately, I wasn't able to convince anyone. Here's £10, hope that pays for many years of future development. Our company depends on it.

**Reply by TodePond** [reply]
Thank you. This will make you rich (in spirit)
